dev/update_users.sh

```diff
@@ -0,0 +1,178 @@
+#!/bin/bash
```

Comment:
All of these users files shouldn't be here. The "maint/users.sql" file is ignored on purpose